src/storage/heap_file.c

```diff
@@ -22428,12 +22694,31 @@ heap_insert_logical (THREAD_ENTRY * thread_p, HEAP_OPERATION_CONTEXT * context,
   PERF_UTIME_TRACKER time_track;
   bool is_mvcc_class;
 
+  LOG_LSA redo_lsa;
+  bool is_supplement = false;
+  LOG_TDES *tdes = NULL;
+
   /* check required input */
   assert (context != NULL);
   assert (context->type == HEAP_OPERATION_INSERT);
   assert (context->recdes_p != NULL);
   assert (!HFID_IS_NULL (&context->hfid));
 
+  if (prm_get_integer_value (PRM_ID_SUPPLEMENTAL_LOG) > 0 && !thread_p->no_supplemental_log)
```

Comment:
How about making an inline function or a macro instead of this line?

I think it is more readable if you leave a comment about the meaning of prm_get_integer_value (PRM_ID_SUPPLEMENTAL_LOG) and thread_p->no_supplemental_log in the inline function or macro.

Reply:
I agree with your opinion. I will make an inline function for better readability.